.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: CI test
+
+on:
+  push:
+    tags:
+      - 'v?[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Concatenate CRDs
+        run: |
+          find config/base/crds/ -name '*.yaml' -maxdepth 1 -exec cat {} + >> kafka-operator.crds.yaml
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Kafka Operator ${{ github.ref }}
+          body: |
+            This release fixes several bug and introduce a couple of new features:
+
+            ## New features:
+            - TODO
```

Comment:
how are we going to populate these TODOs?

Reply:
After the DRAFT release has been created by GHA, you have to edit it on GitHub and publish the release manually.